src/Common/Language.php

```diff
@@ -25,8 +25,8 @@ public static function get()
         $application = 'Backend';
 
         if (Model::has('request')
-            && Model::get('request')->attributes->has('application')
-            && Model::get('request')->attributes->get('application') === 'Frontend'
+            && Model::get('request')->attributes->has('_route')
+            && strpos(Model::get('request')->attributes->get('_route'), 'frontend') === 0
```

Comment:
but to be 100% sure for future changes I made the check case-insensitive 9e0d919